builder/googlecompute/driver.go

```diff
@@ -12,6 +12,9 @@ import (
 // with GCE. The Driver interface exists mostly to allow a mock implementation
 // to be used to test the steps.
 type Driver interface {
+	// CreateDisk creates a persistent disk from the specified config.
```

Comment:
```suggestion
	// CreateDisk creates one or more persistent disks from the specified config.
```

Reply:
The driver command here only creates one disk, each `BlockDevice` represents only one disk, and this function is called for each separate disk we create.